test/test-433-before-001.js

```diff
@@ -0,0 +1,58 @@
+var irc = require('../lib/irc');
+var test = require('tape');
+
+var MockIrcd = require('./mockircd');
+
+test('connect and sets hostmask when nick in use', function(t) {
+    var client, mock, expected;
+
+    t.plan(3);
```

Comment:
This is an extra `plan` call that gets overridden later. I've removed it from other tests